synapse/a2a_compat.py

```
@@ -239,12 +243,12 @@ class ExternalTaskResponse(BaseModel):
 # ============================================================
 
 def create_a2a_router(
-    controller,
+    controller: "TerminalController",
     agent_type: str,
     port: int,
     submit_seq: str = "\n",
-    agent_id: str = None,
-    registry = None
+    agent_id: str | None = None,
+    registry: "AgentRegistry | None" = None
```

Ruff (0.14.10)

Unused function argument: `registry`

(ARG001)